### DISCUSSION TITLE
Beyond Eliza: Finding balance between a forced personality and being more human

### DISCUSSION THREAD

**Discussion by PizBernina** [elizaOS/eliza#819] [Ideas]
I wonder how you see this. Frankly speaking, the social media output of AI agents such as Eliza on X, Telegram etc. are funny when one starts discovering them but then it rather quickly gets somewhat forced. 
Forced as in they are constantly (24/7) in their role letting no space for being human or similar.
Some logic that puts the agents in a **schedule** with **awareness of time, memory and emotional state** that is shared a**cross all social media** would help a lot. Not sure how the eliza framework handles this.
This would make them a bit more unpredictable and thus interesting I'd argue.

Certainly, it is up for debate if that is even where one wants to go. Maybe the 24/7 "acting" is what we prefer. 
Thoughts?

**Comment by fspseva** [comment]
I like this approach and think it will be useful, but also not running your profile entirely autonomous. 

Maybe you could limit the agent to certain number of daily interactions and handle the awareness taking human action.

Then after some time reload tweets and interactions to the model to improve its humanness

**Reply by 73796E7461786572726F72** [reply]
Agree with this approach. I’ve tried letting it run autonomous (on X) and it’s a bit much. So, I have to engage and then it can pick up when someone responds.
